### PR TITLE
Lagoless

### PR DESCRIPTION
**Pull request by oisin100**
Optimizes the way the master controller processes subsystems by having some subsystems prioritizes over others and slowing certain things down(Boms + singularity) so that the other subsystems dont get stuck on something what they shouldent.

Eh, I explained that horribly.

TLDR: You can now bomb the shit out of the station and have the ability to move at the same time.

Credit goes to @MrStonedOne for the actual code.

PS: Sorry Kn0ss0s about pushing the branch to the wrong repo...

### CODE REVIEW

**Comment by oisin100** [comment]
There's probably still a few magic variables floating around from @DavidEdwards old system.

File: html/changelogs/Oisin100-Lagoless.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+#   rscdel (general deleting of nice things)
+#   imageadd
+#   imagedel
+#   spellcheck (typo fixes)
+#   experiment
+#   tgs (TG-ported fixes?)
+#################################
+
+# Your name.  
+author: N3X15
```
Comment (Super3222) [inline html/changelogs/Oisin100-Lagoless.yml]: N3X15?

Comment (oisin100) [inline html/changelogs/Oisin100-Lagoless.yml]: Woops.

**Comment by MrsTonedOne** [comment]
I just pushed some new commits,

mainly adding lag check to contents_explosion, map loader, and a few admin mass edit verbs

**Comment by oisin100** [comment]
> I just pushed some new commits,

Updated, Thank you!

**Comment by DavidEdwards** [comment]
> There's probably still a few magic variables floating around from @DavidEdwards old system

I should have a look before this hits master, looking good so far though!

**Comment by MrsTonedOne** [comment]
also: To explain it better:

We just reserve the final 15% of every byond tick for byond to send out client updates and do its misc processing.

By putting that macro in big for loops, it can just sleep(world.tick_lag) once it hits 85% usage on the current tick, waiting until the next tick.

Also, More improvements, and it just got merged, i tried to keep the commits atomic for easy port updating.

**Comment by oisin100** [comment]
Ported latest updates.
Also fixed a couple of runtimes due to our air system being outdated.

**Pull request merged**